ci/test-container.sh

```diff
@@ -9,6 +9,17 @@ fatal() {
 
 versionid=$(. /usr/lib/os-release && echo $VERSION_ID)
 
+# This allows running this test in a podman container locally by running
+# `SELF_BOOTSTRAP=1 ci/test-container.sh`.
+if [ -n "${SELF_BOOTSTRAP:-}" ]; then
+  rm -rf "$PWD/installtree"
+  make install DESTDIR="$PWD/installtree"
```

Comment:
Non blocking but for the future let's use a subdirectory of `target/` for this type of stuff because it's already in `.gitignore`.

Reply:
Yeah, agree. This is consistent with `tests/vmcheck/install.sh`, but we should update both.